packages/web/index.js

```diff
@@ -100,6 +113,14 @@ module.exports = (neutrino, opts = {}) => {
       .use(EnvironmentPlugin, [options.env]);
   }
 
+  if (options.devtool) {
+    const devtool = options.devtool[process.env.NODE_ENV];
+
+    if (devtool) {
```

Comment:
This will mean that for `false`, `neutrino.config.devtool()` isn't called at all, which for `mode: 'development'` will mean the webpack default of `'eval'` is used, which is probably not what the user intended. I think this should be a `if (process.env.NODE_ENV in options.devtool)` or else `if (devtool !== undefined)`?